README.md

```diff
@@ -0,0 +1,30 @@
+# Potato Farmer
+
+Manages `potato_field` and `potato_plants`. Main way to administrate containers.
+
+## Setup
+
+### Dependencies
+
+There are several dependencies required for the development and deployment of this project.
+The first is Docker Engine and Docker Compose. The following list links to the installation
+instructions for each of these:
+
+* [Docker Engine installation instructions.](https://docs.docker.com/engine/install/)
+* [Docker Compose installation instructions.](https://docs.docker.com/compose/install/linux/)
+
+<!-- TODO: identify and add other dependencies; Python? Rust? -->
```

Comment:
Only Python and Docker are assumed to be dependencies. Everything will be managed in this with python, and Rust will only be launched in containers which will be downloaded on a per-container basis